bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -0,0 +1,149 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% from "firefox/nothing-personal/includes/browser-macro.html" import browser_border %}
+
+{% block page_css %}
+  {{ css_bundle('firefox-nothing-personal') }}
+{% endblock %}
+
+{% block page_title_prefix %}Nothing Personal{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      <h1 class="c-page-header-logo-fx">
+        <img src="{{ static('img/firefox/nothing-personal/firefox-by-mozilla-wordmark.svg') }}" alt="" width="180" height="40" class="c-header-logo">
+        <span>Firefox by Mozilla</span>
+      </h1>
+
+      <div class="cta-group">
+        <p>Feeling browser-curious? We won’t tell. <span aria-hidden="true">>>>>></span></p>
+
+
+        <a href="{{ url('firefox.set-as-default.thanks')}}" class="is-not-default mzp-c-button mzp-t-lg" data-cta-type="button" data-cta-text="Set as default" data-cta-position="{{ position }}">Set as default</a>
+
+        <a href="https://blog.mozilla.org/en/firefox/" class="is-default mzp-c-button mzp-t-lg" data-cta-type="button" data-cta-text="Learn more" data-cta-position="{{ position }}">Learn more</a>
```

Comment:
**issue:** both occurrences of `{{ position }}` are undefined here. Should the value be `nav` to match the download button?

Reply:
I'm going to make the `position` attributes to be `primary` since the header isn't really a nav!